content/sensu-go/5.0/installation/upgrade.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Upgrading to Sensu Go"
+linkTitle: "Upgrade Sensu"
+description: "Guide to upgrading to Sensu Go from Sensu Core"
```

Comment:
I'd suggest changing "Sensu Core" to "Sensu 1.x"

Reply:
We may want to specify a specific version such as 1.3 and above. I believe we do have a cut-off for support?